src/Models/residents.js

```diff
@@ -23,7 +23,7 @@ const residentSchema = new mongoose.Schema({
         type: String,
         required: true,
     },
-    office: {
+    offices: {
```

Comment:
@KristomeisterDev lo mismo debe ser en singular.
```suggestion
    office: {
```

Reply:
correjidos